README.md

```diff
@@ -69,21 +69,22 @@ If the mackerel-agent has already be installed this step can be ignored.
 ## Install mackerel-agent-plugins
 
 Install the plugin pack from either the yum or the apt repository.
+To setup those package repositories, see documentation about installation of mackerel-agent ([rpm](https://mackerel.io/docs/entry/howto/install-agent/rpm) / [deb](https://mackerel.io/docs/entry/howto/install-agent/deb)).
```

Comment:
[nit] [english]
To setup these package repositories, see the documentation regarding the installation of mackerel-agent.